packages/base/links-to-many-component.gts

```diff
@@ -157,6 +157,12 @@ export function getLinksToManyComponent({
       </template>
     };
   } else {
-    return getPluralViewComponent(arrayField, field, format, cardTypeFor);
+    return getPluralViewComponent(
+      arrayField,
+      field,
+      format,
+      cardTypeFor,
+      context
+    );
```

Comment:
@habdelra @jurgenwerk this is the place where the context was not passed down. It probably is a thing with contains-many component tooo

Reply:
Nice find.